easyfix-session/src/acceptor.rs

```diff
@@ -261,6 +261,32 @@ impl<S: MessagesStorage + 'static> Acceptor<S> {
         session.reset(&mut session.state().borrow_mut());
     }
 
+    /// Sneder seq_num getter
+    pub fn next_sender_msg_seq_num(&self, session_id: &SessionId) -> SeqNum {
+        let active_sessions = self.active_sessions.borrow();
+        let Some(session) = active_sessions.get(session_id) else {
+            warn!("reset: session {session_id} not found");
```

Comment:
`reset` looks like copy paste from `fn reset()`

BTW. easyfix uses tracing, you can decorate function with `instrument` macro and it will add context to the log message

```rust
#[instrument(skip_all)]
pub fn next_sender_msg_seq_num(...) {
  ...
  warn!(session_id, "session not found");
}

Reply:
done